tests/test_core.py

```diff
@@ -345,7 +392,6 @@ def get_bullet_library(self, use_cmake):
                             configure_args=configure_args,
                             cache_name_extra=configure_commands[0])
 
-  @also_with_standalone_wasm
```

Comment:
`test_hello_world` is extremely basic, and we test far more interesting things, so it's redundant.

also, we use `test_hello_world` in a few places as a sanity check. it can be confusing to have complicated things like wasm2c happen there.

Reply:
I like the idea of being able to test simple things... now if I want to test the most basic standalone program I need to look for a more complex test?

What do you think about make also_with_standalone_wasm expand to @parameterized and turn the tests into two different tests?

Then test_hello_world_standalone can fail separately to test_hello_world_standalone (I recently made the parameterized thing handle empty string as the name of the variant correctly).

In any case this change is unrelated to wasm2c and doesn't really belong in this PR.

Reply:
Ok, I restored that test now.

We could also use `parametrized`, yeah. I can look into that.